internal/subscription/model.go

```diff
@@ -81,6 +81,11 @@ func (m *Model) GetTopic() string {
 	return m.Topic
 }
 
+// GetSubscriptionTopic returns the topic used for subscription fanout topic
+func (m *Model) GetSubscriptionTopic() string {
+	return topic.GetTopicName(m.ExtractedTopicProjectID, m.ExtractedSubscriptionName)
```

Comment:
Currently, the subscription topic is of form `/projects/topics/<subscription-name>`
Think of the scenario below:
I create a topic `/projects/topics/usecase-1234`
I create a subscription `/projects/topics/usecase-1234`

In this case, the subscription topic would be same as the ingestion topic?

It probably is a good idea to add some prefix/suffix to subscription-topic and ensure in validation that such topics can't be created directly by users.